apple-docs/AppleBranchingScheme.md

```diff
@@ -1,38 +1,45 @@
 # Apple's branching scheme for llvm-project
 
-There are currently three namespaces for branches on
-[github.com/apple/llvm-project](https://github.com/apple/llvm-project):
+This document explains the various important branches in
+[apple/llvm-project](https://github.com/apple/llvm-project),
+how they relate to one another, and how they correspond to branches in the
+[apple/swift](https://github.com/apple/swift) repository.
 
- 1. `llvm.org/*`, for forwarded branches from
-    [github.com/llvm](https://github.com/llvm/llvm-project);
- 2. `apple/*`, for standalone downstream content; and
- 3. `swift/*`, for downstream content that depends on
-    [Swift](https://github.com/apple/swift).
+These are the most important branches to know about:
 
-## Forwarded branches from [github.com/llvm](https://github.com/llvm/llvm-project)
+- [next](https://github.com/apple/llvm-project/tree/next)
+  is aligned with Swift's
+  [next](https://github.com/apple/swift/tree/next) branch.
+- The current stabilization branch
```

Comment:
I'm worried this will quickly become. I would recommend just mentioning the "stable" branch which is aligned with swift's main and keep the reference to update-checkout-config.json to find the exact branch.

Reply:
Fixed.